### ISSUE TITLE
class:list is silently skipped by every class rule in Astro

### ISSUE THREAD

**Issue by abed-daloopa** [shadcn-ui/lint#22]
The class rules read `class` but never `class:list`. Astro uses `class:list` for any conditional or array of classes, so a component that keeps its classes there passes with zero errors.

## Repro

```astro
---
---
<a class="p-[13px]">checked</a>
<a class:list="p-[14px]">not checked</a>
<a class:list={['p-[15px]']}>not checked</a>
```

With `no-arbitrary-values` enabled, the linter reports only line 3:

```
3:10  error  "p-[13px]" hardcodes an off-token value. Use "p-3.25" instead (same value, on the scale)  shadcn/no-arbitrary-values

✖ 1 problem (1 error, 0 warnings)
```

## Cause

`dist/index.js` matches class attributes against one regex:

```js
const CLASS_ATTRIBUTE = /class(name)?s?$/i
```

`class:list` ends in `list` and fails the test. No setting extends the attribute names, so config cannot work around it.

## Why it matters

The failure is silent. The file passes and looks compliant. In my blog, one component held every arbitrary value in the project (`min-w-[70px] px-[5px] py-[10px]`) and reported nothing until I rewrote it as a template literal on `class`.

## Possible fix

Match `class:list` alongside `class`, or let the attribute pattern be configured. `class:list` accepts a string, array, object, or Set, so array and object values need the same handling the rules already apply to `clsx`-style arguments.

## Setup

Astro 7.3.2 with `astro-eslint-parser` 3.1.0, @shadcn/lint 0.1.0, ESLint 10.10.0, @typescript-eslint/parser 8.70.0.

Five of the six rules work correctly on `.astro` files through this parser, which is why the gap is easy to miss.
